src/components/Header/MenusComp/index.tsx

```diff
@@ -59,29 +75,72 @@ const useMenuDataList = () => {
   return list
 }
 
-const MenuItemLink = ({ menu }: { menu: MenuData }) => {
-  const { url, type, name } = menu
+const SubmenuDropdown: FC<{ menu: MenuData }> = ({ children, menu }) => {
   return (
-    <MobileMenuLink href={url} target={type === LinkType.Inner ? '_self' : '_blank'} rel="noopener noreferrer">
-      {name}
-    </MobileMenuLink>
+    <Dropdown
+      overlay={
+        <div className={styles.submenu}>
+          {menu.children?.map(menu => (
+            <a
+              key={menu.name}
+              className={styles.link}
+              href={menu.url}
+              target={menu.type === LinkType.Inner ? '_self' : '_blank'}
+              rel="noopener noreferrer"
+            >
+              {menu.name}
+            </a>
+          ))}
+        </div>
+      }
+      mouseEnterDelay={0}
+      transitionName=""
+    >
+      {children}
+    </Dropdown>
   )
 }
 
 export default memo(({ isMobile }: { isMobile: boolean }) => {
   const menuList = useMenuDataList()
+
   return isMobile ? (
     <MobileMenuItem>
       {menuList
         .filter(menu => menu.name !== undefined)
-        .map(menu => (
-          <MenuItemLink menu={menu} key={menu.name} />
-        ))}
+        .map(menu =>
+          menu.children ? (
+            <SubmenuDropdown key={menu.name} menu={menu}>
+              <MobileMenuLink className={styles.mobileSubmenuTrigger}>
+                {menu.name}
+                <ArrowIcon className={styles.icon} />
+              </MobileMenuLink>
+            </SubmenuDropdown>
+          ) : (
+            <MobileMenuLink
+              key={menu.name}
+              href={menu.url}
+              target={menu.type === LinkType.Inner ? '_self' : '_blank'}
+              rel="noopener noreferrer"
+            >
+              {menu.name}
+            </MobileMenuLink>
+          ),
+        )}
     </MobileMenuItem>
   ) : (
     <HeaderMenuPanel>
       {menuList.map(menu =>
-        menu.type === LinkType.Inner ? (
+        // eslint-disable-next-line no-nested-ternary
+        menu.children ? (
+          <SubmenuDropdown key={menu.name} menu={menu}>
+            {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
+            <a className={classNames('headerMenusItem', styles.submenuTrigger)}>
```

Comment:
Why not use `Link` for navigation

Reply:
This is a trigger, not a link. It doesn't have an href, it just uses an 'a' tag as a carrier. This structure is referenced from the antd example.

Reply:
Got it